plugin/vim/README.md

```diff
@@ -79,7 +79,12 @@ This option requires [Neovim][] 0.5 or newer and [Packer][]
 Add the following line to your `init.lua` file between
 `require('packer').startup(function()` and `end)`:
 
-    use('strager/quick-lint-js', rtp = 'plugin/vim/quick-lint-js.vim')
+    use('strager/quick-lint-js', rtp = 'plugin/vim/quick-lint-js.vim', requires = {'neovim/nvim-lspconfig'})
+
+Then, add the following lines to your `init.lua` file:
+
+    require('lspconfig/quick_lint_js')
+    require('lspconfig').quick_lint_js.setup {}
```

Comment:
I think these instructions should be in a separate section for configuring. Reasons why:

* These instructions are also needed if you install using Vim-Plugged or another plugin manager.
* These instructions are only relevant when using Neovim + nvim-lspconfig.

(I've been meaning to add a section for configuring the ALE plugin (#175).)